tidb-cloud/dev-guide-bi-looker-studio.md

```diff
@@ -25,7 +25,7 @@ To complete this tutorial, you need:
 
 **If you don't have a TiDB Cloud Serverless cluster, you can create one as follows:**
 
-- [Create a TiDB Cloud Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md#step-1-create-a-tidb-cloud-serverless-cluster)
+- [Create a TiDB Cloud Serverless cluster](/develop/dev-guide-build-cluster-in-cloud.md#step-1-create-a-tidb-serverless-cluster)
```

Comment:
Note: I'll update this anchor link by adding `cloud` in https://github.com/pingcap/docs/pull/18744.

Reply:
Updated in https://github.com/pingcap/docs/pull/18744/files#diff-b24124ffe092ce9c9a30a0faf3bb94a789e2fb05999dd6c492e60c5f3b20a1d9.